tests/unit_tests/test_invalid_build.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from magniv.build.build import build
+from tests.unit_tests.fixtures.test_files import (
+    INVALID_KWARG_FILE,
+    INVALID_TASK_KEY_FILE,
+    INVALID_TRIGGER_FILE,
+)
+
+# TODO: merge with TestBuild upon refactor there
+
+
+class TestBuildInvalid:
```

Comment:
This maybe can be pushed off to switching this to `TestBuild` but I think the base class should be in it's own file (test_base_class or something).

Also let's make a method stub here:
```python
class TestBuildBase:
   ...
   
   @pytest.fixture(scope="class")
   def file(self):
       """Method stub for test file"""
       pass
```

Reply:
Moved and added stub, good call. Added a `FIXME` in testbuild to reflect the fact that we should update it to use the base class too in the next pass